docs/element-templates/README.md

```diff
@@ -213,6 +226,14 @@ The resulting properties panel control looks like this:
 
 ![properties panel drop down](field-dropdown.png)
 
+###### Omitted Type
+
+By omitting the `type` configuration the default UI component will be rendered for the respective binding. This behavior is currently only supported for `camunda:inputParameter` and `camunda:outputParameter`. The component will include a toggle `Local Variable Assignment`. When untoggling, the respective `camunda:inputParameter` or `camunda:outputParameter` element will not be created in the BPMN XML.
```

Comment:
It calls 'Local Variable Assignment' for inputs and 'Process Variable Assignment' for outputs. No idea whether we _have to_ mention it for both.

Reply:
True, I will not explicitly include these but rather speak about a generic `toggle`